screen/edit-transaction/src/main/java/com/ivy/transaction/EditTransactionViewModel.kt

```diff
@@ -305,6 +306,11 @@ class EditTransactionViewModel @Inject constructor(
         return transactionAssociatedTags.value
     }
 
+    @Composable
+    private fun getShowTitleSuggestions(): Boolean {
```

Comment:
Edit `getTitleSuggestions()` to return empty list if the feature is disabled. This way we'll also save computation time by not executing the suggestions calculation when the feature is disabled

Reply:
Good idea 😊